notes/design_principles.md

```diff
@@ -0,0 +1,101 @@
+
+## Design Principles for Data Modeling in HyBox
+
+### Use discovery and search requirements as a pragmatic means to determine the needed extent of the modeling efforts
+
+For example, physical description of a postcard is intended only to be recorded and displayed to the user and is not intended to be interoperable in terms of description across objects.  Given the above principle, the physical description can be a simple string with arbitrary content, understandable to the human end-user but not to a machine for comparison.
+
+```json
+{"dc:extent": "3x4 inches"}
+```
+
+is thus acceptable compared to the more complex
+
+```json 
+{
+  "height": {"value": 3, "units": "in"},
+  "width": {"value": 4, "units": "in"}
+}
+```
+
+### Clear separation of Real World and Digital Objects allows easy replacement in different contexts
+
+While it takes slightly more implementation effort, and there is a performance impact of creating and managing more objects in the persistence system, the value of keeping the digital object's identity separate from the real world object's identity enables and promotes later extensions and revisions.  It is not possible for any one system to _a priori_ have models for all possible content types, especially as there are already multiple ontologies and descriptive systems in use within every community.  With a clearly described set of expectations (an API) for what information is required about the RWO, the linked data model and persistence layer can be changed out to accomodate these new, emerging requirements.
```

Comment:
:rocket: This sounds like a good topic to translate into layperson terms for a blog post or FAQ item.